usr/local/sbin/entrypoint.sh

```diff
@@ -56,6 +56,10 @@ else
 fi
 
 main() {
+  # store the fact that we're running the entrypoint
```

Comment:
There's already a sentinel file for a similar purpose at `/run/container-ready` that `is-ready` already checks. Does it not work as intended?

I would not think it's super useful to know if entrypoint specifically is running or not. It's more useful to know that services are actually running, which is the purpose of that sentinel file.